docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -0,0 +1,62 @@
+# BigDL-LLM Installation: GPU
+
+## Quick Installation
+
+Install BigDL-LLM for GPU supports using pip through:
+
+```bash
+pip install --pre --upgrade bigdl-llm[xpu] -f https://developer.intel.com/ipex-whl-stable-xpu
+```
+
+```eval_rst
+.. note::
+
+   The above command will install ``intel_extension_for_pytorch==2.0.110+xpu`` as default. You can install specific ``ipex``/``torch`` version for your need.
+```
+
+## Recommended Requirements
+
+BigDL-LLM for GPU supports has been verified on:
+
+* Intel Arc™ A-Series Graphics
+* Intel Data Center GPU Flex Series
+
+To apply Intel GPU acceleration, there're several steps for tools installation and environment preparation:
+
+* Step 1, only Linux system is supported now, Ubuntu 22.04 is prefered.
+* Step 2, please refer to our [drive installation](https://dgpu-docs.intel.com/driver/installation.html) for general purpose GPU capabilities.
+  ```eval_rst
+  .. note::
+
+    IPEX 2.0.110+xpu requires Intel® oneAPI Base Toolkit's version >= 2023.2.0.
```

Comment:
This note should be moved to step 3.

Reply:
Thank you :) have updated based on comments